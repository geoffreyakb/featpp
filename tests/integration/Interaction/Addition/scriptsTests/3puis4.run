#!/bin/bash 
java Add << EOF
3
4
EOF