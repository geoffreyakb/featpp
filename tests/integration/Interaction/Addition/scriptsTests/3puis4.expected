Entrez un premier entier
4
Entrez un second entier
3
La somme des deux nombres entrÃ©s est 7